oe_auth.services.yml

```diff
@@ -1,7 +1,9 @@
+parameters:
+  pcas.base_url: ~
```

Comment:
We should prefix parameters and service names with the module's name, something like:
```
parameters:
  oe_auth.pcas.base_url: ~
```